sbkeys/generate-aws-sbkeys

```diff
@@ -345,4 +377,3 @@ cp -t "${OUTPUT_DIR}" \
   vendor.crt \
   kms-sign.json \
   config-sign.key \
-  efi-vars.{aws,json}
```

Comment:
Why delete these? They're required to use the profile to register AMIs.

Reply:
Agreed. Seems to be a mistake.